services/jobrunner/secrets-template.env

```diff
@@ -35,3 +35,7 @@ STATA_LICENSE=
 # TELEMETRY
 # honeycomb auth
 OTEL_EXPORTER_OTLP_HEADERS="Authorization=..."
+
+# airlock
+DJANGO_SECRET_KEY=this-is-a-bad-secret-key-please-never-use-in-prod
+AIRLOCK_API_TOKEN=1234567890abcdef
```

Comment:
Hmm, I thought we were going to use JOB_SERVER_TOKEN?

e.g. in docker-compose.yaml

```
env:
    AIRLOCK_API_TOKEN: ${JOB_SERVER_TOKEN}
```

Reply:
we can't do that, because `JOB_SERVER_TOKEN` is set in `02_secret.env` so in my testing this pattern does not work as intended:

```
    env_file:
      - /home/opensafely/config/01_defaults.env
      - /home/opensafely/config/02_secrets.env
      - /home/opensafely/config/03_backend.env
      - /home/opensafely/config/04_local.env
    environment:
      AIRLOCK_API_TOKEN: ${JOB_SERVER_TOKEN}
```

I think we could put this in 02_secrets.env or later (i.e. 03_backend.env), although in terms of its meaning obviously the best place to put this would have been in 01_defaults.env

Reply:
have put it parameterised in `02_secrets.env` - will need to update the files on the test-backend when deploying